app/src/main/java/org/gdg/frisbee/android/appwidget/UpcomingEventWidgetProvider.java

```diff
@@ -92,7 +92,7 @@ public void buildUpdate(final Context context, final AppWidgetManager manager, f
             final RemoteViews views = new RemoteViews(context.getPackageName(), R.layout.widget_upcoming_event);
             Intent mainIntent = new Intent(context, MainActivity.class);
             final PendingIntent pi = PendingIntent.getActivity(context, REQUEST_CODE_LAUNCH_FRISBEE,
-                mainIntent, PendingIntent.FLAG_UPDATE_CURRENT);
+                    mainIntent, PendingIntent.FLAG_UPDATE_CURRENT);
```

Comment:
Hey I think your code style settings is different then the default. We are using the default settings and we have editor config support which should be applied automatically. 

You can change the style setting project by project. Can you choose the default settings for Frisbee. 

Please do not include changes you didn't make. Thank you :)